specification/integration_context.md

```diff
@@ -0,0 +1,91 @@
+<!-- markdownlint-configure-file 
+{ 
+  "MD013": { 
+    "tables": false 
+  } 
+}
+-->
+
+# Integration Context
+
+**Status**: [Experimental](../README.md#versioning-and-status-of-the-specification)
+
+In order to create a more complete picture of a distributed environment, we
+propose the use of additional headers to pass extra context details between
+Cisco observability components. Specifically, we are concerned with passing
+information between OpenTelemetry-based Splunk distributions and AppDynamics
+agents, in both directions.
+
+# Headers
+
+The following defines several new headers and describes their intended use.
+All headers MUST be treated as optional -- peer services will not always
+generate them. These headers SHOULD be treated as opaque values of type
+string.
+
+* `cisco-ctx-acct-id` - Contains the ID of the AppDynamics account.
+* `cisco-ctx-app-id` - Contains the ID of the AppDynamics application.
+* `cisco-ctx-tier-id` - Contains the ID of the AppDynamics tier.
+* `cisco-ctx-bt-id` - Contains the ID of the AppDynamics business transaction (BT).
+* `cisco-ctx-env` - Contains
+  the [deployment.environment.name](https://opentelemetry.io/docs/specs/semconv/attributes-registry/deployment/)
+  resource value from an OpenTelemetry based component.
+* `cisco-ctx-service` - Contains the [service.name](https://opentelemetry.io/docs/specs/semconv/resource/#service)
+  resource value from an OpenTelemetry based component.
```

Comment:
nit: can we use the "standardized/canonical" way of defining headers

```suggestion
* `Cisco-Ctx-Acc-ID` - Contains the ID of the AppDynamics account.
* `Cisco-Ctx-App-ID` - Contains the ID of the AppDynamics application.
* `Cisco-Ctx-Tier-ID` - Contains the ID of the AppDynamics tier.
* `Cisco-Ctx-BT-ID` - Contains the ID of the AppDynamics business transaction (BT).
* `Cisco-Ctx-Env` - Contains
  the [deployment.environment.name](https://opentelemetry.io/docs/specs/semconv/attributes-registry/deployment/)
  resource value from an OpenTelemetry based component.
* `Cisco-Ctx-Service` - Contains the [service.name](https://opentelemetry.io/docs/specs/semconv/resource/#service)
  resource value from an OpenTelemetry based component.
```

Reply:
I don't think it matter much -- but my purely aesthetic preference is to use lowercase. Most libraries/frameworks standardize on lowercase anyway I believe.

Reply:
> Most libraries/frameworks standardize on lowercase anyway I believe.

I am not sure. For sure this is not true for Go :laughing: 
- https://pkg.go.dev/net/http#Header
- https://pkg.go.dev/net/http#CanonicalHeaderKey

> I don't think it matter much 

It does not, but this is how most RFC and docs like https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers follow :wink: 

I won't fight for it, but sometime the difference in "casing" makes it easier for me to "pattern-match" the "type". E.g. when I see `SPLUNK_IS_COOL` I see env var :laughing:

Reply:
Resolving and leaving as lowercase for now. Can revisit if there is increased pressure sometime. I also don't feel super strongly about it.